vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -35,4 +35,5 @@ type Config struct {
 	OnBuild         []string              // ONBUILD metadata that were defined on the image Dockerfile
 	Labels          map[string]string     // List of labels set to this container
 	StopSignal      string                `json:",omitempty"` // Signal to stop a container
+	Policies        map[PolicyType]string // List of policy values
```

Comment:
I am not aware; but including this structure is sufficient to ensure that `docker inspect` will show the policies associated with a container.